lib/test/axe-apca/README.md

```diff
@@ -32,3 +32,19 @@ axe.configure({
     console.log(results);
 });
 ```
+
+### Using custom APCA thresholds
+
+To set custom thresholds for APCA checks, follow these steps:
+
+1. Use `custom` as the first argument when calling `registerAxeAPCA`.
+1. Provide a function as the second argument, optionally accepting `fontSize` and `fontWeight` arguments.
+
+
+```js
+const customConformanceThresholdFn = (fontSize) => {
```

Comment:
it might be worth for documentation purposes to also pass the `fontWeight` as second argument even if we don't use it.